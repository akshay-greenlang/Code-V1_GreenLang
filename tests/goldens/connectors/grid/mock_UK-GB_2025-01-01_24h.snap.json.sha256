fb189dcf914469ab6892354195bae4d915ccdbeebf333e19b00b0a434500dc7c