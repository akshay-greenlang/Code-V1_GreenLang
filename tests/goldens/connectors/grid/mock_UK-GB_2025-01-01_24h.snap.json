{"connector_id":"grid/intensity/mock","connector_version":"0.1.0","payload":{"metadata":{"algorithm":"seasonal_sinusoid","connector":"grid/intensity/mock","data_points":24,"region_offset":190,"seed":"048c82953ec9ae3b5d14290c70b69fd905afef9e6ea219365b07cc143ad4529e","version":"0.1.0"},"region":"UK-GB","resolution":"hour","series":[{"quality":"simulated","ts":"2025-01-01T00:00:00Z","unit":"gCO2/kWh","value":"490"},{"quality":"simulated","ts":"2025-01-01T01:00:00Z","unit":"gCO2/kWh","value":"541.8"},{"quality":"simulated","ts":"2025-01-01T02:00:00Z","unit":"gCO2/kWh","value":"590"},{"quality":"simulated","ts":"2025-01-01T03:00:00Z","unit":"gCO2/kWh","value":"631.4"},{"quality":"simulated","ts":"2025-01-01T04:00:00Z","unit":"gCO2/kWh","value":"663.2"},{"quality":"simulated","ts":"2025-01-01T05:00:00Z","unit":"gCO2/kWh","value":"683.2"},{"quality":"simulated","ts":"2025-01-01T06:00:00Z","unit":"gCO2/kWh","value":"690"},{"quality":"simulated","ts":"2025-01-01T07:00:00Z","unit":"gCO2/kWh","value":"683.2"},{"quality":"simulated","ts":"2025-01-01T08:00:00Z","unit":"gCO2/kWh","value":"663.2"},{"quality":"simulated","ts":"2025-01-01T09:00:00Z","unit":"gCO2/kWh","value":"631.4"},{"quality":"simulated","ts":"2025-01-01T10:00:00Z","unit":"gCO2/kWh","value":"590"},{"quality":"simulated","ts":"2025-01-01T11:00:00Z","unit":"gCO2/kWh","value":"541.8"},{"quality":"simulated","ts":"2025-01-01T12:00:00Z","unit":"gCO2/kWh","value":"490"},{"quality":"simulated","ts":"2025-01-01T13:00:00Z","unit":"gCO2/kWh","value":"438.2"},{"quality":"simulated","ts":"2025-01-01T14:00:00Z","unit":"gCO2/kWh","value":"390"},{"quality":"simulated","ts":"2025-01-01T15:00:00Z","unit":"gCO2/kWh","value":"348.6"},{"quality":"simulated","ts":"2025-01-01T16:00:00Z","unit":"gCO2/kWh","value":"316.8"},{"quality":"simulated","ts":"2025-01-01T17:00:00Z","unit":"gCO2/kWh","value":"296.8"},{"quality":"simulated","ts":"2025-01-01T18:00:00Z","unit":"gCO2/kWh","value":"290"},{"quality":"simulated","ts":"2025-01-01T19:00:00Z","unit":"gCO2/kWh","value":"296.8"},{"quality":"simulated","ts":"2025-01-01T20:00:00Z","unit":"gCO2/kWh","value":"316.8"},{"quality":"simulated","ts":"2025-01-01T21:00:00Z","unit":"gCO2/kWh","value":"348.6"},{"quality":"simulated","ts":"2025-01-01T22:00:00Z","unit":"gCO2/kWh","value":"390"},{"quality":"simulated","ts":"2025-01-01T23:00:00Z","unit":"gCO2/kWh","value":"438.2"}],"unit":"gCO2/kWh"},"provenance":{"connector_id":"grid/intensity/mock","connector_version":"0.1.0","metadata":{"data_points":24,"end":"2025-01-02T00:00:00+00:00","region":"UK-GB","resolution":"hour","start":"2025-01-01T00:00:00+00:00"},"mode":"record","query_hash":"9f4ea4d785f2905bc099a78047592747ef617bacdb29d3b10bcbe479de722c01","schema_hash":"2cf663a8a03d50acb0c1df2fba40940ffb4f63f0d1882a6386a70320e2bc06f6","seed":"048c82953ec9ae3b5d14290c70b69fd905afef9e6ea219365b07cc143ad4529e","snapshot_id":"7bb4f44970bb9050cd0524714028a05973c77307c170fdefb88876562055d6e0","timestamp":"2025-10-09T05:13:40.753214Z"},"schema":{"payload_model":"GridIntensityPayload","provenance_model":"ConnectorProvenance"}}